deploy/kiali/kiali_cr.yaml

```diff
@@ -573,6 +576,7 @@ spec:
 #   component_status: Jaeger info for the Istio Component Status masthead indicator. Only if it is enabled.
 #       app_label: The value of the Jaeger pod(s)'s app label. The name of the app label is defined in the setting 'istio_labels.app_label_name'. It defaults to jaeger.
 #       is_core: Whether the component is core for your deployment. It defaults to false.
+#       namespace: The namespace where Jaeger is installed in. It defaults to the 'istio_namespace' setting.
```

Comment:
You should at least set these four settings to `""` in the defaults/main.yml so the Kiali CR still sets it. The server can default to what it wants if its "" but we should have all defaults explicitly defined (only in rare cases do we leave values completely undefined).